<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfLoop</class>
 <widget class="QWidget" name="ConfLoop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ConfLoop{
background-color: rgb(255, 255, 255);
}
QToolButton{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:10px;
background-color: rgb(255, 255, 255);
}
QLabel{
font: 16pt &quot;文泉驿等宽微米黑&quot;;
color:rgb(15, 41, 68);
}
QGroupBox{
font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(0, 122, 165);
border:2px solid rgb(0, 122, 165);
border-radius:8px;
margin-top:22px;
}
QGroupBox:title{
subcontrol-origin: margin;
left: 5px;
}</string>
  </property>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;文泉驿等宽微米黑&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 122, 165);</string>
   </property>
   <property name="text">
    <string> 回路配置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_1">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>60</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-1</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_2">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>60</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-2</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_3">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>60</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-3</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_4">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>60</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-4</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_5">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>165</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-5</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_6">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>165</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-6</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_7">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>165</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-7</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_8">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>165</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-8</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_9">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>270</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-9</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_10">
   <property name="geometry">
    <rect>
     <x>345</x>
     <y>270</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-10</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_11">
   <property name="geometry">
    <rect>
     <x>555</x>
     <y>270</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-11</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtnLoop_12">
   <property name="geometry">
    <rect>
     <x>765</x>
     <y>270</y>
     <width>141</width>
     <height>80</height>
    </rect>
   </property>
   <property name="text">
    <string>回路-12</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
